spec/spec.md

```diff
@@ -865,7 +871,10 @@ Descriptor Objects_ are composed as follows:
         ****MUST**** be an array consisting of one or more valid URI strings for
         the acceptable credential schemas. A common use of multiple entries in
         the `uri` array is when multiple versions of a credential schema exist
-        and you wish to express support for submission of more than one version.
+        and you wish to express support for submission of more than one version. 
```

Comment:
should remove first person voice if possible

Reply:
`one wishes`

Reply:
`and a Verifier wishes`